build/testing/sauce-labs/test-browsers.js

```diff
@@ -0,0 +1,199 @@
+ */
+const getConfiguration = (phases) => {
+
+    // Parse command arguments and return valid configs
+    try {
+        switch (phases) {
+            
+            case "alpha":
+                return new Configure(Phases.Alpha);
+                
+            case "beta":
+                return new Configure(Phases.Beta);
+                
+            case "release":
+                return new Configure(Phases.Release);
+                
+            default:
+                console.log(chalk.red("Invalid Argument : must be 'alpha', 'beta', or 'release'"));
+                return process.exit(1);
+        }
+
+    } catch(err) {
+        console.error(err)
+    }
+}
+
+// Assign browser configurations to an array
+let browsers = getConfiguration(process.argv[2]);
+
+/**
+ * Run tests on Sauce Labs
+ * @param {string} branchName 
+ */
+function Test(branchName) {
+    console.log("Testing Git branch:", branchName);
+
+    // Execute Selenium/Appium Web Drivers on Sauce Labs for each browser configuration
+    var flows = browsers.map(function (browser) {
+
+        // Setup capabilities
+        let caps = {
+                name: "FAST-DNA MSFT Documentation",
+                build: branchName,
+                tags: "msft-docs",
+                appiumVersion: browser.appiumVersion,
+                platform: browser.platform,
+                platformName: browser.platformName,
+                platformVersion: browser.platformVersion,
+                deviceName: browser.deviceName,
+                orientation: browser.deviceOrientation,
+                browserName: browser.browserName,
+                version: browser.version,
+                screen_resolution: browser.screenResolution,
+                language: browser.language,
+                locale: browser.locale,
+                username: username,
+                password: accessKey,
+                extendedDebugging: true
+        };
+
+        // Setup WebDriver
+        let driver = new Builder()
+                .withCapabilities(caps)
+                .usingServer(remoteHub)
+                .build();
+
+        // Start session and execute test cases
+        driver.getSession().then(async function (session) {
+
+                let sessionId = session.id_;
+
+                try {
+                        await driver.get(domain);
+
+                        // Note: There are many ways to nagivate/find elements on a page
+                        // unfortunately, not all are cross-browser compliant and so
+                        // for example, we can't use xpath. As a result, we're using
+                        // navigation though not necessarily most performant but effective.
+
+                        // After browsers matrix is determined we can fine tune and
+                        // optimize to use other selectors to see what works. If we're
+                        // t8esting mostly on modern browsers this isn't as much a limitation.

```

Comment:
nit: "testing"

Reply:
You don't like 'Tasting' modern browsers?  ;-)  To be fixed.